### DISCUSSION TITLE
CYD Alternative support

### DISCUSSION THREAD

**Discussion by TiggerMoon** [witnessmenow/ESP32-Cheap-Yellow-Display#206] [Q&A]
Hello all! I have a ESP32-3248S035. I am very much a noob when it comes to custom firmwares and flashing, but is there any plans for support for these other boards? Or is there a resource i can use to take some existing firmwares for the CYD (2432S028) and make them work with the aforementioned CYD variant? 

Cheers

**Comment by chupocro** [comment]
I don't have that display but you may take a look at my approach to testing a similar display:

https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/discussions/200#discussioncomment-10078820

Or you can examine these repositories:

https://github.com/ardnew/ESP32-3248S035
https://github.com/rzeldent/esp32-smartdisplay
